src/app/components/notifier/notifier.service.js

```diff
@@ -18,7 +18,7 @@
 
     function info(text) {
       notifications.info("Information", text, {
-        duration: 20000,
+        duration: 5000,
```

Comment:
Reason why I use so big number is to allow user to move mouse and click button, for older users like me it is hard to notice notification, catch mouse and then move it there :)